webservices/common/models/reports.py

```diff
@@ -49,11 +49,11 @@ class CommitteeReports(PdfMixin, BaseModel):
     refunded_other_political_committee_contributions_ytd = db.Column(db.Numeric(30, 2), doc=docs.add_ytd(docs.REFUNDED_OTHER_POLITICAL_COMMITTEE_CONTRIBUTIONS))
     refunded_political_party_committee_contributions_period = db.Column(db.Numeric(30, 2), doc=docs.add_period(docs.REFUNDED_POLITICAL_PARTY_COMMITTEE_CONTRIBUTIONS))
     refunded_political_party_committee_contributions_ytd = db.Column(db.Numeric(30, 2), doc=docs.add_ytd(docs.REFUNDED_POLITICAL_PARTY_COMMITTEE_CONTRIBUTIONS))
-    total_contributions_period = db.Column(db.Numeric(30, 2), doc=docs.add_period(docs.CONTRIBUTIONS))
+    total_contributions_period = db.Column('total_contributions_period', db.Numeric(30, 2), doc=docs.add_period(docs.CONTRIBUTIONS))
```

Comment:
No, they definitely have to be there, at least if you are piping your columns through filters in filters.py, like I did for these columns, here is the stack track when you remove the labels:

```
 
  File "/Users/jonathancarmack/.pyenv/versions/3.5.1/lib/python3.5/site-packages/sqlalchemy/sql/elements.py", line 494, in compile
    return self._compiler(dialect, bind=bind, **kw)
  File "/Users/jonathancarmack/.pyenv/versions/3.5.1/lib/python3.5/site-packages/sqlalchemy/sql/elements.py", line 500, in _compiler
    return dialect.statement_compiler(dialect, self, **kw)
  File "/Users/jonathancarmack/.pyenv/versions/3.5.1/lib/python3.5/site-packages/sqlalchemy/sql/compiler.py", line 392, in __init__
    Compiled.__init__(self, dialect, statement, **kwargs)
  File "/Users/jonathancarmack/.pyenv/versions/3.5.1/lib/python3.5/site-packages/sqlalchemy/sql/compiler.py", line 190, in __init__
    self.string = self.process(self.statement, **compile_kwargs)
  File "/Users/jonathancarmack/.pyenv/versions/3.5.1/lib/python3.5/site-packages/sqlalchemy/sql/compiler.py", line 213, in process
    return obj._compiler_dispatch(self, **kwargs)
  File "/Users/jonathancarmack/.pyenv/versions/3.5.1/lib/python3.5/site-packages/sqlalchemy/sql/visitors.py", line 81, in _compiler_dispatch
    return meth(self, **kw)
  File "/Users/jonathancarmack/.pyenv/versions/3.5.1/lib/python3.5/site-packages/sqlalchemy/sql/compiler.py", line 1602, in visit_select
    text, select, inner_columns, froms, byfrom, kwargs)
  File "/Users/jonathancarmack/.pyenv/versions/3.5.1/lib/python3.5/site-packages/sqlalchemy/sql/compiler.py", line 1694, in _compose_select_body
    for f in froms])
  File "/Users/jonathancarmack/.pyenv/versions/3.5.1/lib/python3.5/site-packages/sqlalchemy/sql/compiler.py", line 1694, in <listcomp>
    for f in froms])
  File "/Users/jonathancarmack/.pyenv/versions/3.5.1/lib/python3.5/site-packages/sqlalchemy/sql/visitors.py", line 81, in _compiler_dispatch
    return meth(self, **kw)
  File "/Users/jonathancarmack/.pyenv/versions/3.5.1/lib/python3.5/site-packages/sqlalchemy/sql/compiler.py", line 1257, in visit_alias
    asfrom=True, **kwargs) + \
  File "/Users/jonathancarmack/.pyenv/versions/3.5.1/lib/python3.5/site-packages/sqlalchemy/sql/annotation.py", line 73, in _compiler_dispatch
    self, visitor, **kw)
  File "/Users/jonathancarmack/.pyenv/versions/3.5.1/lib/python3.5/site-packages/sqlalchemy/sql/visitors.py", line 81, in _compiler_dispatch
    return meth(self, **kw)
  File "/Users/jonathancarmack/.pyenv/versions/3.5.1/lib/python3.5/site-packages/sqlalchemy/sql/compiler.py", line 1602, in visit_select
    text, select, inner_columns, froms, byfrom, kwargs)
  File "/Users/jonathancarmack/.pyenv/versions/3.5.1/lib/python3.5/site-packages/sqlalchemy/sql/compiler.py", line 1699, in _compose_select_body
    t = select._whereclause._compiler_dispatch(self, **kwargs)
  File "/Users/jonathancarmack/.pyenv/versions/3.5.1/lib/python3.5/site-packages/sqlalchemy/sql/visitors.py", line 81, in _compiler_dispatch
    return meth(self, **kw)
  File "/Users/jonathancarmack/.pyenv/versions/3.5.1/lib/python3.5/site-packages/sqlalchemy/sql/compiler.py", line 921, in visit_binary
    return self._generate_generic_binary(binary, opstring, **kw)
  File "/Users/jonathancarmack/.pyenv/versions/3.5.1/lib/python3.5/site-packages/sqlalchemy/sql/compiler.py", line 937, in _generate_generic_binary
    opstring + \
  File "/Users/jonathancarmack/.pyenv/versions/3.5.1/lib/python3.5/site-packages/sqlalchemy/sql/visitors.py", line 81, in _compiler_dispatch
    return meth(self, **kw)
  File "/Users/jonathancarmack/.pyenv/versions/3.5.1/lib/python3.5/site-packages/sqlalchemy/sql/compiler.py", line 624, in visit_column
    raise exc.CompileError("Cannot compile Column object until "
sqlalchemy.exc.CompileError: Cannot compile Column object until its 'name' is assigned.
```